specifyweb/frontend/js_src/lib/components/Molecules/DateElement.tsx

```diff
@@ -40,12 +40,10 @@ function DateElementSafe({
   React.useEffect(() => {
     let timeout: NodeJS.Timeout | undefined;
 
-    function updateRelativeDate() {
+    function updateRelativeDate(): void {
```

Comment:
this function had a bug before actually, since it used `HOUR` rather than `HOUR * MILLISECONDS` before
this refactor would make things less confusing